tests/windows_bindings.rs

```diff
@@ -0,0 +1,44 @@
+// Uncomment this to regenerate bindings
```

Comment:
I'd recommend running this test unconditionally. Then you can ensure that the bindings are always up to date and regressions haven't snuck in. You can use a diff check via CI to ensure the generated bindings haven't diverged from what's committed to your repo.

Reply:
It's definitely a lot more work to maintain this kind of dependency than simply using a Cargo dependency on the `windows-sys` crate. 😊